client-ts/signalr-protocol-msgpack/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@aspnet/signalr-protocol-msgpack",
+  "version": "1.0.0-preview1-t000",
+  "description": "MsgPack Protocol support for ASP.NET Core SignalR",
+  "main": "./dist/cjs/index.js",
+  "browser": "./dist/browser/signalr-protocol-msgpack.js",
+  "module": "./dist/esm/index.js",
+  "typings": "./dist/esm/index.d.ts",
+  "umd_name": "signalR.protocols.msgpack",
+  "directories": {
+    "test": "spec"
+  },
+  "scripts": {
+    "clean": "node ../node_modules/rimraf/bin.js ./dist",
```

Comment:
The `node ../node_modules/.../` syntax is needed because we actually install the tools into the parent directory's `package.json`. If we installed them in this `package.json` we could use `rimraf ../dist`, **but** we'd have copies of them in both the `signalr` and `signalr-protocol-msgpack` folders :(